libexec/tfenv-uninstall

```diff
@@ -28,7 +28,7 @@ else
 fi
 
 [ -n "${version}" ] || error_and_die "Version is not specified"
-version="$(tfenv-list | grep -e "${regex}" | head -n 1)"
+version="$(tfenv-list | sed 's/ //g;s/^*//;s/(.*)//' | grep -e "${regex}" | head -n 1)"
```

Comment:
The regexp for sed is bit hard to understand.
Could you replace it to `sed -E 's/^\* ([0-9\.]+) \(set by .+\)$/\1/g'`
I checked it works expectedly.

```sh
$ echo "* 0.10.7 (set by /opt/tfenv/version)" | sed -E 's/^\* ([0-9\.]+) \(set by .+\)$/\1/g'
0.10.7
```

BTW, I learned sed can include multiple regexps with `;` 😃

Reply:
Ah, sorry I orverlooked that versions which are not the current version are also added ` `. I'll reconsider it.

Reply:
How about `sed -E 's/^(\*| )? //g; s/ \(set by .+\)$//g'` ?

Reply:
If you think it's clearer, I am agree :-)